boilerplate/app/components/Toggle.tsx

```diff
@@ -184,26 +184,29 @@ function Checkbox(props: ToggleInputProps) {
   } = props
 
   const offBackgroundColor = [
-    disabled && colors.palette.neutral300,
+    disabled && colors.palette.neutral400,
```

Comment:
Generally speaking, we should avoid using `colors.palette` unless it's a very one-off situation. Instead, we should have a semantic name, like for example `colors.inactiveElement` or something.

Reply:
@jamonholmgren following up on this. In a situation like this where it's used often within a component, should we create a new semantic name within this context or just use the `colors.border` that's already defined?

Reply:
@cdanwards I honestly am the worst person to ask ... maybe @robinheinze or @yulolimum?

Reply:
Since this is not a border, I would not use `colors.border`. If this is only used in this component and doesn't represent an intentional design choice to make all "inactive elements" this color throughout the app, then I think it's fine to use palette.